contrib/libvpl/module.defs

```diff
@@ -1,11 +1,11 @@
 $(eval $(call import.MODULE.defs,LIBVPL,libvpl))
 $(eval $(call import.CONTRIB.defs,LIBVPL))
 
-LIBVPL.FETCH.url       = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs/oneVPL-2022.0.4.tar.gz
-LIBVPL.FETCH.url      += https://github.com/oneapi-src/oneVPL/archive/refs/tags/v2022.0.4.tar.gz
-LIBVPL.FETCH.sha256    = 7fb0965f1a22344a044840462637f23be0d019bacd011ca2ff179d72dc40064b
-LIBVPL.FETCH.basename  = oneVPL-2022.0.4.tar.gz
-LIBVPL.EXTRACT.tarbase = oneVPL-2022.0.4
+LIBVPL.FETCH.url       = https://github.com/oneapi-src/oneVPL/archive/refs/tags/v2022.1.2.tar.gz
```

Comment:
@sr55 please upload archive to contrib repo

Reply:
Done.

Reply:
Thanks